executor/slow_query.go

```diff
@@ -465,3 +509,151 @@ func ParseTime(s string) (time.Time, error) {
 	}
 	return t, err
 }
+
+type logFile struct {
+	file       *os.File  // The opened file handle
+	begin, end time.Time // The start/end time of the log file
```

Comment:
better use `start` here since the comment and checker...

Reply:
great, done.